apidef/oas/oas_test.go

```diff
@@ -35,3 +35,345 @@ func TestOAS(t *testing.T) {
 
 	assert.Equal(t, api.AuthConfigs, converted.AuthConfigs)
 }
+
+func TestValidateOASObject(t *testing.T) {
+	var validOAS3Definition = []byte(`{
+	  "openapi": "3.0.0",
```

Comment:
Instead of putting it raw can we create a minimum valid go object and use that? Same for invalid one. It will make the test shorter and cleaner